tools/setup-devenv.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -eux
+
+if [ -f /etc/debian_version ]; then
+  sudo apt install -y -V lsb-release
+  distribution=$(lsb_release --id --short | tr 'A-Z' 'a-z')
+  code_name=$(lsb_release --codename --short)
+  case "${distribution}-${code_name}" in
+    debian-bookworm)
+      # For Debian 12
+        sudo apt-get -V -y install wget tar build-essential zlib1g-dev liblzo2-dev libmsgpack-dev libevent-dev libmecab-dev libreadline-dev
+        sudo apt update
```

Comment:
Can we remove them because we don't build Groonga?

Reply:
Ah, we need this coz if you just fresh install Debian or Ubuntu, there is no gcc and necessary libraries.

Reply:
OK. Then how about installing `gcc`, `make` and `pkgconf`?

Reply:
which is covered by `build-essentials`. And people who are not familiar with these things, this script covers everything, I think.

Plus, what if the contributors find out, they need to change not only PGroonga but also Groonga source code? This script will cover that needs too.

Reply:
`build-essential` doesn't depend on `pkgconf`? :<

We can create `setup.sh` in `groonga/groonga` for Groonga contributors and reuse it for PGroonga.